src/crypto/pkcs1.js

```diff
@@ -69,25 +71,24 @@ export default {
      * create a EME-PKCS1-v1_5 padding (See {@link https://tools.ietf.org/html/rfc4880#section-13.1.1|RFC 4880 13.1.1})
      * @param {String} M message to be encoded
      * @param {Integer} k the length in octets of the key modulus
-     * @return {String} EME-PKCS1 padded message
+     * @return {Promise<String>} EME-PKCS1 padded message
```

Comment:
Oooh, should we be specifying when a function returns a promise? I didn't do that anywhere!

Reply:
I think so, but I don't think it's a huge deal. I probably missed it a bunch of times in this PR--maybe we should grep for 'async'.